src/components/Container/index.tsx

```diff
@@ -1,7 +1,7 @@
 import { apiGoogle } from '../../api';
 
 // components
-import HeaderTest from '../Header';
+import Header from '../Header';
 import Search from '../Search';
```

Comment:
is Search it's a box with autocomplete? What search? I don't understand. please, name more descriptionally

Reply:
done